### PR TITLE
Feature/preview package name approval

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Add new feature to approve package name. Package name will be considered as approved if review is approved.

image

### CODE REVIEW

**Comment by maririos** [comment]
Is there a way we can see this in the apiview ui instance?

**Comment by praveenkuttappan** [comment]
Fixes #4099

**Comment by praveenkuttappan** [comment]
> Is there a way we can see this in the apiview ui instance?

Yes. I will deploy it to UAT from this PR.

File: src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs:133]: Is approval date interesting for anything? Looks like we have that for api approvals.

**Review by weshaggard — APPROVED** [review]
Looks good to me. Now we need to update our release pipelines to understand the new status code and also call it for the first betas.

File: src/dotnet/APIView/APIViewWeb/Pages/Shared/_ReviewBadge.cshtml
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/Pages/Shared/_ReviewBadge.cshtml:8]: In which cases there won't be an approver? and what does that mean?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Pages/Shared/_ReviewBadge.cshtml:8]: There are many reviews for same package. We show package name as approved when review itself is approved. It's possible that another review is in approved state for same package. We won't have approver name in this case. I will see If I can populate approver name in this case.

File: src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs:133]: It will only be useful to show as tooltip or for an eventual use. I will add that field so we will have that data in case required in the future. thanks

File: src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:464]: nit: 
```suggestion
            review.IsPackageNameApproved = true;
```

File: src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:1]: Did you intend to remove this file?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:1]: Nope. It's not removed. It got formatted differently. Let me see If I can revert this formatting change. :(

**Pull request merged**
